From 28c0fe7248f5022195f55608c9b746fa45b8981a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:11 -0500
Subject: [redhat] kabi: Adding symbol _raw_write_lock_irq (kernel/spinlock.c)

Message-id: <26563ec09d4acdd94a8aa5d03380562081fcb737.1416318767.git.poros@redhat.com>
Patchwork-id: 100129
O-Subject: [PATCH RHEL7.1 06/63] kabi: Adding symbol _raw_write_lock_irq (kernel/spinlock.c)
Bugzilla: 1070274
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    Reader/writer locks. We need them as well.
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 27/80] kabi: Adding symbol _raw_write_lock_irq (kernel/spinlock.c)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:27 +0100)
 Description:
 ============
 Reader/writer locks.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/_raw_write_lock_irq
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/_raw_write_lock_irq

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/_raw_write_lock_irq b/redhat/kabi/kabi-module/kabi_ppc64/_raw_write_lock_irq
new file mode 100644
index 0000000..43718cf
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/_raw_write_lock_irq
@@ -0,0 +1,2 @@
+#1-
+0x0e8f30f6 _raw_write_lock_irq vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/_raw_write_lock_irq b/redhat/kabi/kabi-module/kabi_x86_64/_raw_write_lock_irq
new file mode 100644
index 0000000..925a7a9
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/_raw_write_lock_irq
@@ -0,0 +1,2 @@
+#1-
+0xa3a9637d _raw_write_lock_irq vmlinux EXPORT_SYMBOL
-- 
1.7.1